scripts/check-npm.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-if [[ `npm -v` != 3* ]]
-  then
-    npm install --global npm
-fi
```

Comment:
Well, we want to support LTS version right? The current one is `6`. I guess we can also support node `4`. Using node `6` by default comes with `npm@3` included, although if we keep using `yarn` as dependency manager it doesn't really matter.

So we can support node `4` as well, I don't see a problem for now ;)